src/array/groupArrToNestedArr.js

```diff
@@ -0,0 +1,12 @@
+export default (arr: any[], key: string) =>
```

Comment:
Why the weird abbreviations? 
groupArrToNestedArr
groupArrayToNestedArray

Did you really save that much? :)